anaconda.py

```diff
@@ -304,10 +304,13 @@ def setup_environment():
         log.info("Using updates in /tmp/updates/ from %s", opts.updateSrc)
 
     # warn users that they should use inst. prefix all the time
-    for arg in depr:
-        stdout_log.warning("Boot argument '%s' is deprecated and will be removed in the future. "
-                           "Please use 'inst.%s' instead.",
+    for arg in removed_no_inst_args:
+        stdout_log.warning("The '%s' kernel boot argument detected. "
+                           "Did you want to use 'inst.%s' instead?",
```

Comment:
Sorry, now I see what you meant. How about:

Kernel boot argument '%s' detected. Did you want to use 'inst.%s' for the installer instead?

Reply:
Fixed.